### PR TITLE
Previous activity table

### PR DESCRIPTION
**Pull request by mgwalker**
### This pull request changes...
- previous activity table will exist

### New backend needs
- data model for collecting previous activity approved/actual data (#694)

### This pull request is ready to merge when...
- [x] Tests have been updated (and all tests are passing)
- [x] This code has been reviewed by someone other than the original author
- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented

### This feature is done when...
- [x] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

**Comment by mgwalker** [comment]
The styling isn't quite perfect with the design but I was spending too much time fiddling to try to get it right.  @lauraponce what do you think about creating another issue to refine the styling on this table to get it just right?

File: web/src/containers/ApdPreviousActivityTable.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+          ))}
+          <th colSpan="2" className={`border-right ${borderClass(2)}`}>
+            Federal + state<br />Grand total computable
+          </th>
+        </tr>
+        <tr>
+          <th />
+          {Object.values(programs).map((name, i) => (
+            <Fragment key={name}>
+              <th className={`bg-${colors[i]}-light ${borderClass(i)}`}>
```
Comment (brendansudol) [inline web/src/containers/ApdPreviousActivityTable.js]: these dynamic classes (which I've done a bunch of) feel quite messy to me; fine for now but I'm making a mental note - and actual note here :) - to revisit and clean up.

File: web/src/containers/ApdPreviousActivityTable.js [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+                    label={`actual state share for ${program}, FFY ${year}`}
+                    hideLabel
+                    value={previousActivityExpenses[year][program].stateActual}
+                    onChange={handleChange(year, program, 'stateActual')}
+                  />
+                </td>
+              </Fragment>
+            ))}
+            <td className={`bg-${colors[2]}-light ${borderClass(2)}`}>
+              {formatMoney(27000)}
```
Comment (brendansudol) [inline web/src/containers/ApdPreviousActivityTable.js]: should this be hardcoded?

File: web/src/containers/ApdPreviousActivityTable.js [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+            </td>
+            <td>{formatMoney(3000)}</td>
```
Comment (brendansudol) [inline web/src/containers/ApdPreviousActivityTable.js]: should this number be hardcoded?

Comment (mgwalker) [reply web/src/containers/ApdPreviousActivityTable.js]: Oh, whops.  Nooope.

**Comment by NAretakis** [comment]
@mgwalker looks good. But the totals are not adding up, should they?

**Comment by mgwalker** [comment]
Updated to compute all the totals values.  Took the opportunity to extra labels into the resource file too.

**Comment by jeromeleecms** [comment]
@mgwalker Both column headers say HIT and HIT (vs HIE) - and yes, the total for HIT+HIE column is the sum of the previous items.

Few additional items (@NAretakis may be we need to discuss?)
1) If we have an HIT and HIE table, shouldn't we have a MMIS table as well?

2) Math works now (I heard computers are good at that!), but can we also have the state share auto-populated as a suggestion based on what the states put in at the 90% share? Or maybe good for MVP and we just need to add that as part of the front end validation.

3) Are the years hard coded? If we keep it this way, may be we should include some additional help text to tell the users to only go back as far as the previous APD (that way we are not making the states go grab data from something that closed a couple years ago when they really don't need to do that). 

4) (Styling Comment) Was it a styling thing with the Federal Share (newline) 90% FFP and State Share (no newline) 10% FFP? If we want to be consistent with our letters -- Federal Share (new line) 90% <-- centered in the column header. Happy to table that for the new styling issue. 

5) (Editorial comment) Last column: Federal + S(can we capitalize "S" for consistency)tate Grand Total Computable

**Comment by mgwalker** [comment]
(Fixed the HIE/HIT headers.)

2. We can do that.  I'd suggest making that another issue and prioritizing, though.  I have questions about how we'd do the math here - Do we still have input boxes for both state and federal pieces, and as they fill in one, we compute the other?  Do we only have input boxes for the state share and always compute the federal?  Only input the federal share and always compute the state?  Etc.

3. Years are based on current FFY and previous two, per https://github.com/18F/cms-hitech-apd/issues/652#issuecomment-397450794, for now.

4. Whoops, those are both supposed to have newlines!  Messed up the resource file.

5. Fixed.

**Pull request merged**